branding/README.md

```diff
@@ -0,0 +1,18 @@
+# Spacebar Brand Guidelines
+
+Those are the guidelines for the Spacebar branding usage.
+
+## Colors
+- Primary: <img src="./colors/primary.svg" width="12px" height="12px"> #0185ff 
+- Secondary: <img src="./colors/secondary.svg" width="12px" height="12px">  #000115
+
+## Typography
+### Logo and Headings
+**New Hero (Adobe)**
```

Comment:
maybe instead of the `(Adobe), just a link to the adobe font?

Reply:
[New Hero](https://fonts.adobe.com/fonts/new-hero)

Reply:
Done! Also did it for Inter.